cannon/mipsevm/open_mips_tests/maketests.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+import os
+import sys
+import tempfile
+from capstone import *
+from elftools.elf.elffile import ELFFile
+md = Cs(CS_ARCH_MIPS, CS_MODE_32 + CS_MODE_BIG_ENDIAN)
+
+def maketest(d, out):
+  with tempfile.NamedTemporaryFile() as nf:
+    path = "/Users/kafka/fun/mips/mips-gcc-4.8.1/bin/"
```

Comment:
This path is hardcoded and seems wrong?

Reply:
Yes, legacy from Geohot, this script has never worked for me, at least it serves as some documentation.